packages/api-client/src/conversation/ConversationAPI.ts

```diff
@@ -201,6 +201,16 @@ export class ConversationAPI {
     return response.data;
   }
 
+  public async getConversationGuestLinkFeature(conversationId: string): Promise<{status: 'enabled' | 'disabled'}> {
```

Comment:
What do you think about moving `{status: 'enabled' | 'disabled'}` to its own type?

Reply:
Sure, will do